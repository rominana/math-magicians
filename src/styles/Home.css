
.home-section {
  margin: 0 auto;
  width: 80%;
  color: white;
  padding: 8rem 6rem;
}

h2.home-title {
  font-size: 2.5rem;
  color: white;
  padding-bottom: 30px;
}

h3.home-title {
  font-size: 2rem;
  color: white;
}

.home-p {
  margin: 2rem 0;
  font-size: 0.8rem;
  color: white;
}
