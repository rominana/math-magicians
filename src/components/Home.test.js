import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home';

it('Renders Home component correctly', () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
