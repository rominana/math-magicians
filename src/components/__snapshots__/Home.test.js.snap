// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar tests Renders Navbar component  1`] = `
Array [
  <nav
    className="navbar"
  >
    <h1>
      <a
        className="logo"
        href="/home"
        onClick={[Function]}
      >
        Math Magicians
      </a>
    </h1>
    <ul>
      <li>
        <a
          href="/home"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/calculator"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          href="/quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>,
  " ",
]
`;

exports[`Renders Home component 1`] = `
<div
  className="home-section"
>
  <h2
    className="home-title"
  >
    Math Magicians
  </h2>
  <h3
    className="home-second-title"
  >
    Welcome to our page!
  </h3>
  <p
    className="home-p"
  >
    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
  </p>
  <p
    className="home-p"
  >
    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
  </p>
</div>
`;

exports[`Renders Home component correctly 1`] = `
<div
  className="home-section"
>
  <h2
    className="home-title"
  >
    Math Magicians
  </h2>
  <h3
    className="home-second-title"
  >
    Welcome to our page!
  </h3>
  <p
    className="home-p"
  >
    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
  </p>
  <p
    className="home-p"
  >
    Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
  </p>
</div>
`;
